x-pack/plugins/security_solution/cypress/test_plans/detection_response/prebuilt_rules/prebuilt_rules_install_update_workflows.mdx

```diff
@@ -0,0 +1,180 @@
+# Rule Immutability / Customization
+
+## Test Plan for 2nd Milestone of Customizing prebuilt detection rules
+
+### Useful information
+
+Ticket: [https://github.com/elastic/security-team/issues/1974](https://github.com/elastic/security-team/issues/1974)
+
+**Assumptions**
+- The current test plan is only for Milestone 2 of the Rule Immutability/Customization feature to be released in 8.9. It does not pretend to cover any scenario for past or future milestones. Scenarios and flows are sensitive to change in future Milestones.
+- Below scenarios only apply to prebuilt rules.
+- Most of our users are on the 7.17.x version, that’s why the 8.x version is specified on scenarios, because this TestPlan is considering a minimum version of 8.x.
+- The rule Customization feature should be available to users on the Basic license and higher.
+
+
+### Scenarios
+
+### Notifications
```

Comment:
One note about notifications: the scenarios below talk about callouts, but for this milestone we have removed them until further notice.
See:
- the PR: https://github.com/elastic/kibana/pull/160269
- the ticket to reintroduce them in the future: https://github.com/elastic/kibana/issues/160270

So for now we should concentrate on the notifications that do exist:

- For installation:
    - the Add Elastic Rule button that appears instead of the Rules Management table when there are no rules installed
    - the Add Elastic Rules button that is always at the top of the Rules Management page and shows in a badge how many rules are available for install.
- For upgrade:
    -   The "Rules Updates" tab that shows in a badge how many rules are available for update.

Reply:
Synced with @vgomez-el: 👍

Reply:
@jpdjere Agreed. We can write scenarios using generic "notification" or "CTA" terminology, and then clarify what kinds of notifications we will have in 8.9.0 and what will be added in the next versions.